src/patternfly/components/LabelGroup/label-group.scss

```diff
@@ -34,6 +34,11 @@
   --pf-c-label-group__list-item--MarginRight: var(--pf-global--spacer--xs);
   --pf-c-label-group__list-item--MarginBottom: var(--pf-global--spacer--xs);
 
+  // Add label
+  --pf-c-label-group__add--MarginLeft: var(--pf-global--spacer--xs);
+  --pf-c-label-group__add--MarginRight: var(--pf-global--spacer--xs);
+  --pf-c-label-group__add--MarginBottom: var(--pf-global--spacer--xs);
```

Comment:
To expand on this @jenny-s51, we define vars (or try to anyways 😂) in the order they're used in the CSS. Since the shorthand for `margin/padding/border-width`/etc goes `margin: [top] [right] [bottom] [left];` and if we were to define those properties individually in the css, they would go like this to satisfy the linter `margin-top: [...]; margin-right: [...]; margin-bottom: [...]; margin-left: [...];`, we define the vars in the same order.

Reply:
Thank you for clarifying @mcoker !